default_warehouse_from_sale_team/i18n/es.po

```diff
@@ -86,51 +149,79 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
 msgid "Not matter sales team will leave the user to see all the stock picking types"
 msgstr "No importa los equipos de ventas le permitirá al usuario ver todos los tipos de albarán"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,comment:default_warehouse_from_sale_team.group_manager_default_journal
+msgid "Not matter sales teams will leave the user to see all the Journal"
```

Comment:
allow the user to see all journals, regardless of sales teams

Reply:
Done